spec/amp-actions-and-events.md

```diff
@@ -433,6 +453,27 @@ event.response</pre></td>
   </tr>
 </table>
 
+### amp-accordion <a name="amp-accordion-1"></a>
+
+<table>
+  <tr>
+    <th>Action</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td><code>toggle(section=STRING)</code></td>
+    <td>Toggles the <code>expanded</code> and <code>collapsed</code> states of <code>amp-accordion</code> sections. When called with no arguments, it toggles all sections of the accordion. To specify a specific section, add the <code>section</code> argument and use its corresponding <code>id</code> as the value.</td>
```

Comment:
Minor: example code snippets may be more concise.

Reply:
Done.